sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/arithmetic.scala

```diff
@@ -367,8 +410,16 @@ case class Remainder(left: Expression, right: Expression)
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(a, b) - Returns the positive modulo",
-  extended = "> SELECT _FUNC_(10,3);\n 1")
+  usage = "_FUNC_(expr1, expr2) - Returns the positive modulo.",
+  extended = """
+    Arguments:
+      expr1 - a numeric expression.
+      expr2 - a numeric expression.
+
+    Examples:
+      > SELECT _FUNC_(10, 3);
```

Comment:
(added in https://github.com/apache/spark/pull/15513/commits/5163a871929268865f321de8b861911ac7b0249c#diff-1516b10738479bbe190fb4e239258473R429)